test/marketing_event_test.rb

```diff
@@ -46,4 +46,23 @@ def test_count_marketing_events
     marketing_events_count = ShopifyAPI::MarketingEvent.get(:count)
     assert_equal 2, marketing_events_count
   end
+
+  def test_add_engagements
```

Comment:
that makes sense, and im not sure if checking equality of returned fields in the body is any different from the current implementation so this is good to go 👍